tokio/src/runtime/task/join.rs

```diff
@@ -45,6 +46,49 @@ doc_rt_core! {
     /// # }
     /// ```
     ///
+    /// Explicit JoinHandle<T> specification for a spawned task that does not return a value.
+    /// This is parameterized over `()`:
+    ///
+    /// ```
+    /// use tokio::task;
+    ///
+    /// # async fn doc() {
+    /// let join_handle: task::JoinHandle<()> = task::spawn(async {
+    ///     println!("I return nothing.")
+    /// });
+    /// # }
+    /// ```
+    ///
+    /// Explicit JoinHandle<T> specification for a spawned task that returns a value:
```

Comment:
Can you swap this with the example before, such that `JoinHandle<i32>` comes before `JoinHandle<()>`? Also, more active voice in the sentence.